src/core_plugins/input_control_vis/public/components/editor/control_editor.js

```diff
@@ -0,0 +1,104 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+export class ControlEditor extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isEditorCollapsed: true
+    };
+  }
+
+  handleToggleControlVisibility() {
+    this.setState(prevState => (
+      {  isEditorCollapsed: !prevState.isEditorCollapsed }
+    ));
+  }
+
+  renderEditor() {
+    return (
+      <div>
+        <div className="kuiSideBarFormRow">
+          <label className="kuiSideBarFormRow__label">
+            Label
+          </label>
+          <div className="kuiSideBarFormRow__control kuiFieldGroupSection--wide">
+            <input
```

Comment:
Please label this `input` with the above `label` (see [accessibility style guide](https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#label-every-form-element)) using `htmlFor` in React.